### PR TITLE
layout refactor

### PR DESCRIPTION
**Pull request by demianbrecht**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -115,27 +92,26 @@
...
-        <div class="copyright">
-          © Copyright 2016 Salesforce.com, inc. All rights reserved. Various trademarks held by their respective owners. Salesforce.com, inc. The Landmark at One Market, Suite 300, San Francisco, CA 94105, United States
-        </div>
-      </footer>
-    </div>
+      </div>
+
+      <nav class="slds-p-around--medium"><h2>Want to work with us?</h2><a href="https://www.salesforce.com/tech">salesforce.com/tech</a></nav>
+
+      <div class="slds-text-align--center slds-p-around--xx-large light">
```
Comment (fernomac) [inline public/index.html]: Open source stuff we use? Logos we think look pretty? Not at all opposed (or even related to this change), but might be worth mentioning why these are here?

**Comment by fernomac** [comment]
:+1:

File: public/index.html (hunk 1 above)
Comment (demianbrecht) [inline public/index.html]: Yeah, was trying to think of a good way to call that out while keeping aesthetics (not that I'm a designer at all ;)).

FWIW, you can check out the result of the changes here: http://demianbrecht.github.io/salesforce.github.io/

**Comment by ongardie-sfdc** [comment]
I'm going to resolve the conflict and merge this, because I think people were in agreement to do that. (Maybe y'all fell asleep? I'm in GMT+2.)

**Comment by ongardie-sfdc** [comment]
Merged this into master. I don't know what this src branch is for (if you don't either, then I suggest deleting it).

**Comment by demianbrecht** [comment]
@ongardie-sfdc `src` *was* intended to be the working directory, publishing to master. That said, there's no real reason for it right now (there's no pipeline other than copying files), so I'm happy to remove it until we actually need something like that (although I'm not expecting to for this page).

**Pull request closed without merging**